pkg/sql/opt/optbuilder/select.go

```diff
@@ -577,8 +578,8 @@ func (b *Builder) addCheckConstraintsForTable(tabMeta *opt.TableMeta) {
 
 	// Find the non-nullable table columns.
```

Comment:
Add parenthetical to the comment, saying that mutation cols can be NULL during backfill, so they should be excluded.